### PR TITLE
New package: ft2-clone-1.25

### PR DESCRIPTION
**Pull request by rc-05**
Modern clone of the popular music tracker software Fast Tracker 2.

### CODE REVIEW

File: srcpkgs/ft2-clone/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ft2-clone'
+pkgname=ft2-clone
+version=1.21
+revision=1
+archs="~*-musl"
```
Comment (abenson) [inline srcpkgs/ft2-clone/template:5]: Can't be patched for `musl`?

File: srcpkgs/ft2-clone/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ft2-clone'
+pkgname=ft2-clone
+version=1.21
```
Comment (abenson) [inline srcpkgs/ft2-clone/template:3]: 1.25 is out

File: srcpkgs/ft2-clone/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/ft2-clone/template:5]: Ah, its just missing libfts, that's an easy fix

File: srcpkgs/ft2-clone/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+homepage="https://16-bits.org/ft2.php"
+distfiles="https://github.com/8bitbubsy/${pkgname}/archive/v${version}.tar.gz"
+checksum=2f83768e119112696345e743a54584bb95e97d47a60d215f1ac0602a51974416
+
+do_install() {
+	vbin ${wrksrc}/release/other/ft2-clone
+}
+
+post_install() {
+	vlicense LICENSE
```
Comment (abenson) [inline srcpkgs/ft2-clone/template:20]: This can all be done in `do_install()`

File: srcpkgs/ft2-clone/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/ft2-clone/template:5]: You can drop the `archs` line if add:

```
if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
	makedepends+=" musl-fts-devel"
	configure_args+=" -DCMAKE_EXE_LINKER_FLAGS=-lfts"
fi
```

Before `do_install()`

File: srcpkgs/ft2-clone/template (hunk 3 above)
Comment (rc-05) [reply srcpkgs/ft2-clone/template:21]: Done :smile:

File: srcpkgs/ft2-clone/template (hunk 2 above)
Comment (rc-05) [reply srcpkgs/ft2-clone/template]: Changed version, commit name and PR name to reflect version

File: srcpkgs/ft2-clone/template (hunk 1 above)
Comment (rc-05) [reply srcpkgs/ft2-clone/template]: Done, now it should compile also for the `musl` platforms

**Pull request closed without merging**